images/apko/main.tf

```diff
@@ -0,0 +1,37 @@
+terraform {
+  required_providers {
+    apko = { source = "chainguard-dev/apko" }
+  }
+}
+
+variable "target_repository" {
+  description = "The docker repo into which the image and attestations should be published."
+}
+
+module "latest" {
+  source = "../../tflib/publisher"
+
+  target_repository = var.target_repository
+  config            = file("${path.module}/configs/latest.apko.yaml")
```

Comment:
Just learning... why do we still need this config path defined in `images/apko/image.yaml` if we're defining it for Terraform now?

Reply:
@luhring because (for now) we are still using `monopod` for the matrix.  Once we switch to invoking the "mega module" directly (without `-target=module.apko`) we can delete that file entirely.

Reply:
Got it! Thanks!